atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleOverflowWriteTable.java

```diff
@@ -63,22 +63,24 @@ private OracleOverflowWriteTable(
             ConnectionSupplier conns,
             OverflowSequenceSupplier sequenceSupplier,
             OracleTableNameGetter oracleTableNameGetter,
+            OraclePrefixedTableNames oraclePrefixedTableNames,
             TableReference tableRef) {
         this.config = config;
         this.conns = conns;
         this.overflowSequenceSupplier = sequenceSupplier;
         this.oracleTableNameGetter = oracleTableNameGetter;
-        this.oraclePrefixedTableNames = new OraclePrefixedTableNames(config, conns, oracleTableNameGetter);
+        this.oraclePrefixedTableNames = oraclePrefixedTableNames;
         this.tableRef = tableRef;
     }
 
     public static OracleOverflowWriteTable create(
             OracleDdlConfig config,
             ConnectionSupplier conns,
             OracleTableNameGetter oracleTableNameGetter,
+            OraclePrefixedTableNames oraclePrefixedTableNames,
             TableReference tableRef) {
         OverflowSequenceSupplier sequenceSupplier = OverflowSequenceSupplier.create(conns, config.tablePrefix());
-        return new OracleOverflowWriteTable(config, conns, sequenceSupplier, oracleTableNameGetter, tableRef);
+        return new OracleOverflowWriteTable(config, conns, sequenceSupplier, oracleTableNameGetter, oraclePrefixedTableNames, tableRef);
```

Comment:
See above, but do we actually need to take the `OraclePrefixedTableNames` as an arg to create?
```
return new OracleOverflowWriteTable(config,
        conns,
        sequenceSupplier,
        oracleTableNameGetter,
        new OraclePrefixedTableNames(oracleTableNameGetter),
        tableRef);
```